install.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json.schemastore.org/install",
+  "workers": [
+    {
+      "src": "./dist/main.js"
+    }
+  ],
+  "options": {
+    "properties": {
+      "API_KEY": {
+        "title": "API Key",
+        "type": "string",
+        "required": true,
+        "description": "Key associated with your README project.",
+        "placeholder": "API Key"
+      },
+      "ROUTES": {
+        "title": "Routes",
+        "type": "array",
```

Comment:
Alternatively, instead of accepting an array we could just accept a single ROUTE? This would more closely mirror the instantiation of the middleware:

```
app.use('/api/*', readme.metrics('api-key')
```

Reply:
The current cloudflare worker implementation accepts as many routes as you'd like. Additionally, where you add the wrapper in express to any route, you can only install 1 instance of a worker on your website at a time.

Reply:
Yup! Users can potentially apply this to multiple subdomains, something akin to `*.example.com/`. We should be able to support this, so examining the whole URI is important.